src/index.html

```diff
@@ -14,6 +14,27 @@
 </head>
 <body>
 
+<section class="search-field_large">
+  <label>
+    <input
+            type="text"
+            placeholder="&quot;Try Los Angeles&quot;"
+            data-qa="big"
+            data-qa="keypress"
+            class="search large-input">
```

Comment:
Is good to write attribute in new line if you have more than 3 or some long attributes, but will be better to follow the next formatting:

```html
<input
  type="text"
  placeholder="&quot;Try Los Angeles&quot;"
  data-qa="big"
  data-qa="keypress"
  class="search large-input"
>
```

Keep indentation the same as in other places and move close tag arrow(`>`) into a new line. Like an object or array literals